README.md

```diff
@@ -65,7 +66,7 @@ command in order to really clean everything and rebuild from scratch:
 
     % make distclean
 
-This will clean: jemalloc, lua, hiplaceholderkv, linenoise and other dependencies.
+This will clean: jemalloc, lua, hiredis, linenoise and other dependencies.
```

Comment:
Do we need to fork hiredis repository at some point?

Reply:
+1 to keep the original name. It is functional